.github/workflows/want-gpus.yml

```
@@ -0,0 +1,17 @@
+name: I love GPUs!
+
+on:
+  pull_request:
+
+jobs:
+  multilication-table-on-gpu:
+    runs-on: $${ matrix.os }}
```

```suggestion
    runs-on: ${ matrix.os }}
```

Just saw webhook logs from this:
```
2021-10-01T15:34:01 2021-10-01T15:34:01.513Z    19038ee9-23af-4def-8c41-e9313648d63e    INFO    Sending message (Group 2): {"id":3769006105,"repositoryName":"pytorch","repositoryOwner":"pytorch","eventType":"workflow_job","installationId":<redacted>,"runnerLabels":["$${ matrix.os }}"],"callbackUrl":"https://github.com/pytorch/pytorch/runs/3769006105"}
```